web/src/routes.rs

```diff
@@ -18,5 +19,6 @@ pub fn routes(app_state: AppState) -> axum::Router {
             "/insert_articles",
             post(users::add_url::insert_article_handler),
         )
+        .route("/article/:article_id", get(article_detail))
```

Comment:
new route to display the summary of an article on the web page

Reply:
I reason why I have `insert_articles` is because clippy is yelling at me for amount of characters is not satisfying for it 😂

Reply:
Can we do `/artciles/:article_id` just to match the `/articles` plural route above this?